hubspot/connection/__init__.py

```diff
@@ -204,6 +204,7 @@ def _require_successful_response(response):
             raise exception_class(
                 error_data['message'],
                 error_data['requestId'],
+                error_data,
```

Comment:
Hi @frankoid, sorry, I didn't mean the `failureMessages`, I actually meant the `error_data`, right now I don't have much time to deal with this (since we're in the middle a sprint right now) but I can tell you what I have in mind so you can think about this too.

My "issue" with passing `error_data` is that it would give too much low level data to `hubspot-connection` clients, which is not great since it's a different level of abstraction. I see the ideal situation for this being a structure that we can pass to clients with the same level of abstraction as `message` and `code` (meaning, in a proper structure) so that the client can deal with it accordingly, without having to parse through undocumented Hubspot data. Now, in order to do this, of course I'd need some time to check the format and all the error possibilities that Hubspot could get so that I can find an appropriate Schema for it.

Once I have some time, that's what I'll do. I'll let you know once I resume working on your pull request.